sql/2020/19_Compression/distribution_of_text_compression_lighthouse.sql

```diff
@@ -0,0 +1,15 @@
+#standardSQL
+#Distribution of Text Based Compression Lighthouse Scores
+SELECT 
+  _TABLE_SUFFIX AS client,
+  JSON_EXTRACT_SCALAR(report, "$.audits.uses-text-compression.score") AS text_compression_score,
+  COUNT(0) AS num_pages,
```

Comment:
Also add a field for total pages
```suggestion
  COUNT(0) AS num_pages,
  SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS total,
```